src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Controls/DataGridCell.cs

```diff
@@ -964,6 +964,28 @@ protected override void OnPreviewKeyDown(KeyEventArgs e)
         /// </summary>
         protected override void OnKeyDown(KeyEventArgs e)
         {
+            if ((Keyboard.Modifiers & ModifierKeys.Alt) == ModifierKeys.Alt)
+            {
+                DataGridLength updatedWidth = new DataGridLength();
+
+                if (e.Key == Key.Right)
+                {
+                    updatedWidth = new DataGridLength(this.Column.ActualWidth + _columnWidthStepSize);
```

Comment:
As the coding-style said, we avoid `this.` unless absolutely necessary.

See https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/coding-style.md

Reply:
Pushed changes as per the comments - adapted PascalCasing.